### ISSUE TITLE
string-strip-html index.d.ts incorrectly defines tag attributes as an array of strings

### ISSUE THREAD

**Issue by gh-andre** [codsen/codsen#38]
**Package's name**

`string-strip-html`

**Describe the bug**

`index.d.ts` defines `Tag` with `attributes` defined as an array of strings:

https://github.com/codsen/codsen/blob/main/packages/string-strip-html/types/index.d.ts

```javascript
interface Tag {
  attributes: string[];
  lastClosingBracketAt: number;
  lastOpeningBracketAt: number;
  slashPresent: number;
  leftOuterWhitespace: number;
  onlyPlausible: boolean;
  nameStarts: number;
  nameContainsLetters: boolean;
  nameEnds: number;
  name: string;
}
```
The examples treat it as an object (e.g. in https://codsen.com/os/string-strip-html/examples/#retain-href-and-link-label) and printing it shows that it's an array of objects:
```javascript
  attributes: [
    {
      nameStarts: 3,
      nameEnds: 8,
      equalsAt: 8,
      name: 'style',
      valueStarts: 10,
      valueEnds: 20,
      value: 'color: red'
    }
  ],
``` 

**To Reproduce**

Run this code
```javascript
stripHtml('<p style="color: red">ABC</p>', {
                        skipHtmlDecoding: true,
                        cb: cbObj => console.log(cbObj.tag)
                    });
```
, to see this output:
```javascript
{
  attributes: [
    {
      nameStarts: 3,
      nameEnds: 8,
      equalsAt: 8,
      name: 'style',
      valueStarts: 10,
      valueEnds: 20,
      value: 'color: red'
    }
  ],
  lastOpeningBracketAt: 0,
...
```
**Expected behavior**

The `attributes` array should be defined as an array of attribute objects they are, so it's possible to use this in TypeScript without casting these strings to objects.

**Comment by revelt** [comment]
hi Andre, Good spot! Sorry about that! I'll fix.

**Comment by revelt** [comment]
I've fixed it over weekend, will attempt to release in the evening today. Sorry it took longer than normal, the last month was very busy.

**Comment by revelt** [comment]
should be fixed in newly-released `9.1.7`

**Comment by gh-andre** [comment]
@revelt Thank you
